src/app/components/kudos-challenge-completed/kudos-challenge-completed.service.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+    "use strict";
+    angular.module("myApp.components.completedChallenges")
+        .factory("CompletedChallengesService", CompletedChallengesService);
+
+    CompletedChallengesService.$inject = [
+        "Challenges"
+    ];
+
+    function CompletedChallengesService(challengesBackend) {
+        var service = {
+            completedChallenges: CompletedChallenges
+        };
+        return service;
+
+        function CompletedChallenges() {
+            return challengesBackend.completedChallenges();
```

Comment:
The naming of the method doesn't say anything. Function/method names should be self explanatory and usually begin with a verb, so this method is probably a `getCompletedChallenges`? `fetchCompletedChallenges`. A developer should know what the method does just by looking at the name. As Robert C. Martin(Uncle Bob - creator of the "Clean Code" series) said - function and variables names are the best documentation.